operator/pkg/cli/kafka.go

```diff
@@ -256,6 +256,15 @@ func (kc *KafkaClient) getPipelineStatus(pipelineSpec string) (*scheduler.Pipeli
 	return res, nil
 }
 
+func getPipelineNameFromHeaders(headers []kafka.Header) (string, error) {
+	for _, header := range headers {
+		if header.Key == "pipeline" {
```

Comment:
@cliveseldon could we have multiple pipeline headers? i.e. in the case of wiring up two pipelines together?

Reply:
Don't think so